pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -513,7 +711,11 @@ def phase_str(phi: float) -> str:
         for coords in ch_data.target:
             targets = list(ch_data.target[coords])
             tgt_strs = [str(q) for q in targets]
-            tgt_txt_y = max_amp * 1.1 - 0.25 * (len(targets) - 1)
+            if isinstance(ch_obj, DMM):
+                tgt_strs = ["⚄"]
```

Comment:
I think the font for the symbol could be increased, it's a bit small

Reply:
![image](https://github.com/pasqal-io/Pulser/assets/61453516/5b687d97-66f1-4ea9-96ad-2913c9675187)

Reply:
Instead of this 
![image](https://github.com/pasqal-io/Pulser/assets/61453516/3ee24141-895d-48e5-98e1-c3f25f6b466e)

Reply:
Looks better!